packages/hoppscotch-app/components/lenses/renderers/composables/useResponseBody.ts

```diff
@@ -0,0 +1,20 @@
+import { computed, Ref } from "@nuxtjs/composition-api"
+import { HoppRESTResponse } from "~/helpers/types/HoppRESTResponse"
+
+export default function useResponseBody(response: HoppRESTResponse): {
+  [key: string]: Ref<any>
+} {
+  const responseBodyText = computed(() => {
+    if (response.type === "loading" || response.type === "network_fail")
```

Comment:
```suggestion
    if (
      response.type === "loading" ||
      response.type === "network_fail" ||
      response.type === "script_fail" ||
      response.type === "fail"
    )
```
Please add the new two more possible type checks in all such instances as well:
- `RawLensRenderer.vue`
- `RawLensRenderer.vue`
- `ResponseMeta.vue`
- `lenses.ts`

Reply:
Sure @liyasthomas ! Could you re-review it?